src/vuestic-theme/vuestic-components/vuestic-breadcrumbs/VuesticBreadcrumbs.vue

```diff
@@ -33,6 +55,7 @@
 <style lang='scss' scoped>
   .vuestic-breadcrumbs {
     height: $breadcrumbs-height;
+    padding-bottom: 25px;
```

Comment:
ok, revert this back, let .vuestic-breadcrumbs be inside the widget with .no-padding .no-v-padding